src/containers/__Routing/Routing.js

```diff
@@ -42,8 +40,7 @@ const RoutingBody = (props) => {
                                     Login using the Pin button on the login screen on any PC or Mobile device.<br /><br />
                                     Otherwise make sure to setup your face login to login hands free!<br /><br />
                                     {/*This will only be shown to you once, and will be deleted within ? days, so remember to change your email and password!<br /><br />*/}
-                                    <h5>Do you want to set up Face Login? Say: </h5><h4 style={{color: "blue"}}>🎤 Mirror mirror go to my account</h4>
```

Comment:
was line 45 intended to be deleted.  Does seem like it should have been for this change.

Reply:
I'm not sure I did not intentionally initiate that change.